### PR TITLE
chore: Make sharp to an optional dependency

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Addresses https://github.com/appium/appium/issues/18780

### CODE REVIEW

File: lib/commands/actions.js [hunk 1]
```diff
@@ -2,17 +2,24 @@ import androidHelpers from '../android-helpers';
...
 const dragStepsPerSec = 40;
+let _sharp;
 
 const commands = {};
 const helpers = {};
 const extensions = {};
 
+function requireSharp() {
+  if (!_sharp) {
+    _sharp = require('sharp');
```
Comment (jlipps) [inline lib/commands/actions.js:18]: do we want to wrap this in error handling with a nice error message so the user knows what to do if this fails?

**Review by jlipps — APPROVED** [review]

File: lib/commands/actions.js (hunk 1 above)
Comment (jlipps) [reply lib/commands/actions.js:18]: or maybe this `requireSharp` function can even just be part of appium support? isn't sharp already an optional dep there?

**Review by jlipps — APPROVED** [review]

File: lib/commands/actions.js (hunk 1 above)
Comment (mykola-mokhnach) [reply lib/commands/actions.js]: it is already there

File: lib/commands/actions.js [hunk 2]
```diff
@@ -409,7 +408,7 @@ helpers.getScreenshotDataWithAdbExecOut = async function getScreenshotDataWithAd
     throw new Error('Screenshot returned no data');
   }
 
-  return sharp(stdout);
+  return Buffer.from(stdout, 'ascii');
```
Comment (boneskull) [inline lib/commands/actions.js:411]: this doesn't look right.  `stdout` is already a `Buffer`.

**Pull request merged**